### DISCUSSION TITLE
sms support , is it possible?

### DISCUSSION THREAD

**Discussion by peterlymo** [Security-Onion-Solutions/securityonion#8717] [Unsupported Versions]
is it possible to shift those alerts to sms, lets say i need to be alerted when there crucial alert via sms, any idea or alternatively

**Comment by isaacgolding** [comment]
An option might be to use email -> SMS Gateways.    Most modern Cellular carriers have their own free to use gateways built.  For example here are the verizon gateways:   number@vtext.com (SMS) number@vzwpix.com (MMS)   so you would just send an email to   5551234567@vtext.com and it shows up as a text message moments later.

**Reply by peterlymo** [reply]
Thanks, not sure cellular carriers for my country has this services, i was thinking to have a program that will fetch a new alert then filter and send to my number using bulk-sms, no idea yet how fetch the real time alerts

**Comment by weslambert** [comment]
There is no direct SMS support at the moment, but you might be able to do something similar to what was mentioned above.

**Comment by ghost** [comment]
Using Amazon SNS, you can send SMS. Amazon SNS is supported with Elastalert.
